docs/docs/telemetry.mdx

```diff
@@ -0,0 +1,123 @@
+---
+id: telemetry
+sidebar_label: Rasa Telemetry
+title: Rasa Telemetry
+---
+
+Rasa uses telemetry to report anonymous usage information. This information is
+essential to help improve Rasa Open Source for all users.
+For the team working on Rasa Open Source it is important to understand how
+the product is used. It allows us to properly prioritize our research efforts
+and feature development.
+
+You will be notified about the telemetry reporting when running Rasa Open Source
+for the first time.
+
+## How to opt-out
+
+You can opt out of telemetry reporting at any time using
+```bash
+rasa telemetry disable
+```
+
+or by setting `RASA_TELEMETRY_ENABLED=false` as an environment variable.
+
+If you want to enable reporting again, you can use
+```bash
+rasa telemetry enable
+```
+
+## Why do we use telemetry reporting?
+
+**Anonymous** telemetry data allow us to prioritize our research efforts
+and feature development based on usage. We want to collect aggregated
+information on usage and reliability so that we can ensure a high-quality product.
+
+So how will we use the reported telemetry data? Here are some examples
+what we use the data for:
+
+- We will be able to know which languages, pipelines and policies are used.
+  This will enable us direct our research efforts towards projects
+  that have the most impact on text and dialogue handling.
```

Comment:
i'm not sure this sentence makes sense - do we mean something like "This will enable us to direct our research efforts towards text and dialogue handling projects that will have the biggest impact for our users"?